app/globals.css

```diff
@@ -1,6 +1,34 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import "decanter/src/css/index.css";
```

Comment:
Ideally, this would be just `@import "decanter"` and then we create and allow 'deep imports' with `@import "decanter/utilities"` or `@import "decanter/components"` etc.

Reply:
That sounds like a good plan. I'm thinking that we can have a few presets, like one without all the form stuff. Will definitely talk to you more.